regression/Makefile

```diff
@@ -32,7 +35,13 @@ SYSCALL_TEST_DIR := $(INITRAMFS)/opt/syscall_test
 .PHONY: all
 all: build
 
-$(INITRAMFS)/lib/x86_64-linux-gnu: | $(VDSO_DIR)
+$(INITRAMFS):
+	@mkdir -p $@
+	@if [ -e $(INIT_FILE) ]; then \
+		cp $(INIT_FILE) $@; \
```

Comment:
It is inappropriate to put the init script for benchmark under the root directory. From its name ("init"), one cannot tell that the file is for benchmark. Reorganize the code to make benchmark-related files to more proper locations.

Reply:
Move it to `benchmark` now.